src/main/java/co/zecko/retailer/client/httpClient/BaseClient.java

```diff
@@ -48,7 +49,7 @@ public BaseClient() {
 
         ExecutorService executorService = Executors.newFixedThreadPool(threadCount);
         this.httpClient = HttpClient.newBuilder().executor(executorService).build();
-        this.objectMapper = new ObjectMapper();
+        this.objectMapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
so you are saying to create a new class `X` which will initiate or create the instance of `ObjectMapper` and rather than calling `ObjectMapper` everytime in future we will call `X`, is that correct?